sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/streamingLimits.scala

```diff
@@ -50,8 +50,8 @@ case class StreamingGlobalLimitExec(
   override def validateAndMaybeEvolveStateSchema(
       hadoopConf: Configuration, batchId: Long, stateSchemaVersion: Int):
     List[StateSchemaValidationResult] = {
-    val newStateSchema = List(StateStoreColFamilySchema(StateStore.DEFAULT_COL_FAMILY_NAME,
-      keySchema, valueSchema))
+    val newStateSchema = List(StateStoreColFamilySchema(StateStore.DEFAULT_COL_FAMILY_NAME, 0,
```

Comment:
in all these places, prob better to name the arg ?

`keySchemaId = 0`

and

`valueSchemaId = 0` ?